bodyfixed_quat: 0.8579405470257437,-0.0,-0.0,0.5137489832293272
